azurelinuxagent/ga/exthandlers.py

```diff
@@ -236,32 +236,11 @@ def __init__(self, protocol):
         self.log_process = False
 
         self.report_status_error_state = ErrorState()
-        self.get_artifact_error_state = ErrorState(min_timedelta=ERROR_STATE_DELTA_INSTALL)
 
     def run(self):
-        self.ext_handlers, etag = None, None
-        try:
-            self.ext_handlers, etag = self.protocol.get_ext_handlers()
-            self.get_artifact_error_state.reset()
-        except Exception as e:
-            msg = u"Exception retrieving extension handlers: {0}".format(ustr(e))
-            detailed_msg = '{0} {1}'.format(msg, traceback.extract_tb(get_traceback(e)))
-
-            self.get_artifact_error_state.incr()
-
-            if self.get_artifact_error_state.is_triggered():
-                add_event(AGENT_NAME,
-                          version=CURRENT_VERSION,
-                          op=WALAEventOperation.GetArtifactExtended,
```

Comment:
i think GetArtifactExtended is not used anymore and can be deleted

Reply:
Good catch! removed it